rocketpy/Flight.py

```diff
@@ -1315,14 +1314,14 @@ def uDot(self, t, u, postProcessing=False):
                 compStreamVzBn = compStreamVzB / compStreamSpeed
                 if -1 * compStreamVzBn < 1:
                     compAttackAngle = np.arccos(-compStreamVzBn)
+                    cLift = abs(aerodynamicSurface[1](compAttackAngle))
```

Comment:
This seems perfectly correct, but I am curious about the use of `abs` here. Isn't `compAttackAngle` always positive?

Reply:
On second thought, I think your decision to use `abs` makes the code more robust, making sure the lift force direction is not affected by `cLift`.

Reply:
I was in doubt if `compAttackAngle` were always positive and, as the coefficient must always be positive, abs was added. But at principle it isnt't really necessary

Reply:
What do you suggest?